apps/spruce/src/pages/adminSettings/AdminSaveButton.tsx

```diff
@@ -0,0 +1,58 @@
+import { useMutation } from "@apollo/client";
+import Button from "@leafygreen-ui/button";
+import {
+  SaveAdminSettingsMutation,
+  SaveAdminSettingsMutationVariables,
+} from "gql/generated/types";
+import { SAVE_ADMIN_SETTINGS } from "gql/mutations";
+import { useAdminSettingsContext } from "./Context";
+import { formToGqlMap } from "./tabs/transformers";
+import { FormToGqlFunction } from "./tabs/types";
+
+export const AdminSaveButton = () => {
+  const { checkHasUnsavedChanges, getChangedTabs, getTab } =
+    useAdminSettingsContext();
+  const hasUnsavedChanges = checkHasUnsavedChanges();
+
+  const [saveAdminSettings] = useMutation<
+    SaveAdminSettingsMutation,
+    SaveAdminSettingsMutationVariables
+  >(SAVE_ADMIN_SETTINGS, {});
```

Comment:
here, in the `onCompleted` / `onError` hook, let's dispatch a toast. This gives feedback to the user that something has happened.

[example](https://github.com/evergreen-ci/ui/blob/b198a3352056a621997cffbf88ea3f9115d26bf7/apps/spruce/src/components/SetPriority/index.tsx#L71-L81)